src/printer.rs

```diff
@@ -92,6 +92,12 @@ impl Printer {
                 writeln!(self.out).unwrap();
                 writeln!(self.out, "running {} test{}", num_tests, plural_s).unwrap();
             }
+            FormatSetting::Json => writeln!(
+                self.out,
+                "{{ \"type\": \"suite\", \"event\": \"started\", \"test_count\": {} }}",
```

Comment:
Can you use raw string literals for all strings containing a `"`? I think it's way easier to read. I.e.

```suggestion
                r#"{{ "type": "suite", "event": "started", "test_count": {} }}"#,
```